plugins/dell-dock/dell-dock.quirk

```diff
@@ -138,3 +141,9 @@ FirmwareSizeMax=0x80000
 Flags = skips-restart,require-ac,dual-image
 Icon = thunderbolt
 InstallDuration = 46
+DellDockBlobMajorOffset = 0x400a
+DellDockBlobMinorOffset = 0x4009
+DellDockBlobVidMajorOffset = 0x4221
+DellDockBlobVidMinorOffset = 0x4220
+DellDockBlobPidMajorOffset = 0x4223
+DellDockBlobPidMinorOffset = 0x4222
```

Comment:
I think actually all of these keys either need to be hardcoded in the source file or moved to the unprobed device.  The reason is that you won't have the GUID added until later on, so the keys won't exist yet.

Since this plugin is really **only supporting** Goshen ridge XHCI (and we don't know if follow on some day works the same way), I think hardcoding as `#define` is my preference.